samples/python-flask/13.core-bot/config.py

```diff
@@ -2,13 +2,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-""" Bot Configuration """
```

Comment:
May want to add this back in so pylint doesn't complain.

Reply:
Done!